lib/rex/parser/openvas_nokogiri.rb

```diff
@@ -112,8 +112,8 @@ def end_element(name=nil)
             @state[:proto] = @text.split('(')[1].split('/')[1].split(')')[0]
             record_service unless @state[:name].nil?
           elsif @text.index('/')
-            @state[:port] = @text.split('/')[0]
-            @state[:proto] = @text.split('/')[1]
+            @state[:port] = @text.split('/')[0].strip
```

Comment:
Happy path is meant to have no whitespace:
![image](https://user-images.githubusercontent.com/60357436/94665164-b6b48080-0303-11eb-9fdc-e65d89b4be5e.png)

This import has whitespace:
![image](https://user-images.githubusercontent.com/60357436/94665239-caf87d80-0303-11eb-8d59-e38a11174b06.png)

Which later caused an import error:
```
[-] **Error while running command db_import: Validation failed: Port must be an integer**
```